identity_storage/Cargo.toml

```diff
@@ -21,12 +21,15 @@ identity_document = { version = "=0.7.0-alpha.6", path = "../identity_document",
 identity_iota_core = { version = "=0.7.0-alpha.6", path = "../identity_iota_core", default-features = false, optional = true }
 identity_verification = { version = "=0.7.0-alpha.6", path = "../identity_verification", default_features = false }
 iota-crypto = { version = "0.23", default-features = false, features = ["blake2b", "ed25519", "random"], optional = true }
+iota-sdk = { version = "1.0", default-features = false, features = ["tls", "client", "stronghold"], optional = true }
+iota_stronghold = { version = "2.0", optional = true }
```

Comment:
Let's please follow our codebase convention and use `default-features = false` and activate what we need.